internal/namespace/manager.go

```diff
@@ -216,10 +216,29 @@ func (nm *namespaceManager) Init(ctx context.Context) (err error) {
 }
 
 func (nm *namespaceManager) initNamespace(name string, ns *namespace) error {
+	stored, err := ns.plugins.Database.Plugin.GetNamespace(nm.ctx, name)
+	switch {
+	case err != nil:
+		return err
+	case stored != nil:
+		stored.RemoteName = ns.remoteName
+		stored.Description = ns.description
+		// TODO: should we check for discrepancies in the multiparty contract config?
```

Comment:
Should we do this and return an error? At a minimum, maybe we add a log statement that we're overwriting stored config.